tests/basics/string_format.py

```diff
@@ -63,6 +63,16 @@ def test(fmt, *args):
 test("{:^20}", "foo")
 test("{:<20}", "foo")
 
+# regression of boolean formatting as int
+print('%c' % False)
+test('{:d}', False)
+test('{:f}', False)
+test('{:20}', False)
+print('%c' % True)
```

Comment:
These 2 tests that use string modulo formatting will need to go in `tests/basics/string_format_modulo.py`

Reply:
I'll move them

Reply:
Moved the floating point ones to `float/string_format.py` as well.